doc/source/whatsnew/v1.1.0.rst

```diff
@@ -585,6 +585,7 @@ I/O
   unsupported HDF file (:issue:`9539`)
 - Bug in :meth:`~DataFrame.to_parquet` was not raising ``PermissionError`` when writing to a private s3 bucket with invalid creds. (:issue:`27679`)
 - Bug in :meth:`~DataFrame.to_csv` was silently failing when writing to an invalid s3 bucket. (:issue:`32486`)
+- :func:`read_parquet` now supports an s3 directory (:issue:`26388`)
```

Comment:
you are closing 2 issues? can you list both

Reply:
sure done